include/libtorrent/alert_types.hpp

```diff
@@ -1849,7 +1856,7 @@ namespace libtorrent
 		add_torrent_alert(aux::stack_allocator& alloc, torrent_handle h
 			, add_torrent_params const& p, error_code ec);
 
-		TORRENT_DEFINE_ALERT_PRIO(add_torrent_alert, 67)
+		TORRENT_DEFINE_ALERT_PRIO(add_torrent_alert, 67, alert_priority_realtime)
```

Comment:
yes. that is the case already, right? All alerts that are posted in response to a call into libtorrent are supposed to be prioritized. They aren't subject to the mask, but should still not be likely to be dropped because of a full queue